mypy/checkstrformat.py

```diff
@@ -449,6 +448,24 @@ def perform_special_format_checks(
                     call,
                     code=codes.STRING_FORMATTING,
                 )
+        a_type = get_proper_type(actual_type)
+        if isinstance(a_type, NoneType):
```

Comment:
This could be cleaner:
```suggestion
        if isinstance(get_proper_type(actual_type), NoneType):
```